charts/operator/templates/deployment.yaml

```
@@ -19,6 +19,10 @@ spec:
       securityContext:
         seccompProfile:
           type: RuntimeDefault
+      imagePullSecrets:
+        {{- range .Values.image.pullSecrets }}
+        - name: {{ . }}
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)